src/server/conn_context.h

```diff
@@ -117,6 +118,11 @@ class ConnectionContext : public facade::ConnectionContext {
       : facade::ConnectionContext(stream, owner) {
   }
 
+  ConnectionContext(Transaction* tx)
+      : facade::ConnectionContext(nullptr, nullptr), transaction{tx} {
+    delete Inject(new facade::CapturingReplyBuilder{});
```

Comment:
"delete Inject" - what's this?

Reply:
facade::ConnectionContext creates a new reply builder by default, need to disable it (writing delete was just quicker)